tests/entry-test.js

```diff
@@ -0,0 +1,53 @@
+'use strict';
+
+var fs = require('fs');
+var expect = require('chai').expect;
+var Entry = require('../lib/entry');
+
+require('chai').config.truncateThreshold = 0;
+
+describe('Entry', function() {
+  describe('.fromStat', function() {
+    beforeEach(function() {
+      fs.mkdirSync('fixture');
+    });
+
+    afterEach(function() {
+      fs.rmdirSync('fixture');
```

Comment:
lets use `fs-extra` + `fs.removeSync` here, so always correctly cleanup.

Currently, if as assertion fails before an inner `unlink` / `rmdir`, this will also fail, potentially causing a cascading test failure and the fixture folder will need to be cleaned up manually.